tests/unit/Specification/InPathTest.php

```diff
@@ -25,9 +25,27 @@ class InPathTest extends \PHPUnit_Framework_TestCase
     private $fixture;
 
     /**
-     * Initializes the fixture for this test.
+     * @covers ::__construct
+     * @covers ::isSatisfiedBy
+     * @covers ::<private>
+     * @dataProvider validDirnames
+     * @uses Flyfinder\Path
      */
-    public function setUp()
+    public function testExactMatch()
+    {
+        $absolutePath = 'absolute/path/to/file.txt';
+        $spec = new InPath(new Path($absolutePath));
+        $this->assertTrue($spec->isSatisfiedBy([
+            'type' => 'file',
+            'path' => $absolutePath,
+            'dirname' => $absolutePath,
+            'filename' => 'file',
+            'extension' => 'txt',
+            'basename' => 'file.txt'
+        ]));
+    }
+
+    protected function useWildcardPath()
```

Comment:
Unless this function is used in a subclass I recommend making the visibility private. The project's recommendations are to use final classes and private methods unless there is a clear use-case to make it more visible. This will ensure a stable API and allow us to be flexible in which methods we change without breaking BC